File: src/main/java/com/rover12421/csdn/java2kt/java/_1_CharNumberAttrib.java
package com.rover12421.csdn.java2kt.java;

import java.io.ByteArrayOutputStream;
import java.io.IOException;

/**
 * Created by rover12421 on 8/10/16.
 */
public class _1_CharNumberAttrib {
    public static void main(String[] args) {
        char ch = 'a';
        try (
                ByteArrayOutputStream baos = new ByteArrayOutputStream()
        ) {
            baos.write(ch);
        } catch (IOException e) {
            e.printStackTrace();
        }
    }
}

File: src/main/kotlin/com/rover12421/csdn/java2kt/kt/_1_CharNumberAttrib.kt
package com.rover12421.csdn.java2kt.kt

import java.io.ByteArrayOutputStream
import java.io.IOException

/**
 * Created by rover12421 on 8/10/16.
 */
object _1_CharNumberAttrib {
    @JvmStatic fun main(args: Array<String>) {
        val ch = 'a'
        try {
            ByteArrayOutputStream().use { baos -> baos.write(ch.toInt()) }
        } catch (e: IOException) {
            e.printStackTrace()
        }
    }
}